packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -262,7 +276,12 @@ export const start = async (zcf, privateArgs, baggage) => {
 
   const publicFacet = augmentPublicFacet(
     harden({
+      /** @param {Brand<'nat'>} collateralBrand */
       getBidInvitation(collateralBrand) {
+        /**
+         * @param {ZCFSeat} zcfSeat
+         * @param {import('./auctionBook.js').BidSpec} bidSpec
```

Comment:
`bidSpec` comes from outside the contract (i.e. from a presumed attacker), and `bidProposalShape` doesn't guarantee that `bidSpec` is of type `BidSpec`. It only checks `{}`, whatever that means.